examples/pytorch_imagenet_resnet50.py

```diff
@@ -169,7 +169,8 @@ def train(epoch):
             loss.backward()
             batch_loss += loss
             train_accuracy.update(accuracy(output, target))
-            if (batch_idx + 1) % args.backward_passes_per_step == 0:
+            if ((batch_idx + 1) % args.backward_passes_per_step == 0 or
+                (batch_idx + 1) >= len(train_loader)):
```

Comment:
@alsrgv Shall we restrict users to set-up ``backward_passes_per_step`` to be a factor of the total number of training examples? If the value is not set correctly, the ``step()`` warning would appear.

Reply:
Good idea!